docs/debugging/debugging.md

```diff
@@ -0,0 +1,158 @@
+# Debugging
+
+This section describes procedures which may be used to debug `ropensci-review-bot` problems.
+The most common problem is a failure of the bot to respond.
+This section describes how to diagnose and resolve that problem, with solutions to other problems also generally found by following similar procedures. 
+
+## The Heroku service
+
+The bot itself is hosted on Heroku, which is the first point of diagnosis.
+The logs for the bot will reveal whether commands were successfully received.
+These logs can be viewed by logging in to [heroku](https://heroku.com), clicking on "radiant-garden", then under "More" on the upper right, clicking "Logs".
```

Comment:
```suggestion
These logs can be viewed by 

- logging in to [heroku](https://heroku.com), clicking on "radiant-garden", then under "More" on the upper right, clicking ["Logs"](https://devcenter.heroku.com/articles/logging#view-logs-with-the-heroku-dashboard). This will allow you to see the current log, from the moment you open it. You can open the log this way, launch a command from a GitHub software-review issue and see it appear in the log.

- using the [Heroku CLI](https://devcenter.heroku.com/articles/heroku-cli) on your machine: [logging in](https://devcenter.heroku.com/articles/heroku-cli#get-started-with-the-heroku-cli) and then running [`heroku logs`](https://devcenter.heroku.com/articles/logging#view-logs-with-the-heroku-cli).
```